Source/Tests/Replica/SceneSynchronization.cpp

```diff
@@ -465,27 +465,27 @@ TEST_CASE("Ownership is consistent on server and on clients")
     {
         Node* node = Tests::SpawnOnServer<BehaviorNetworkObject>(serverScene, prefab, "Unowned Node");
 
-        auto object = node->GetDerivedComponent<NetworkObject>();
+        auto object = node->FindComponent<NetworkObject>(ComponentSearchFlag::Self | ComponentSearchFlag::Derived);
         REQUIRE(object->GetNetworkMode() == NetworkObjectMode::Standalone);
     }
     {
         Node* node = Tests::SpawnOnServer<BehaviorNetworkObject>(serverScene, prefab, "Owned Node 0");
 
-        auto object = node->GetDerivedComponent<NetworkObject>();
+        auto object = node->FindComponent<NetworkObject>(ComponentSearchFlag::Self | ComponentSearchFlag::Derived);
         object->SetOwner(sim.GetServerToClientConnection(clientScenes[0]));
         REQUIRE(object->GetNetworkMode() == NetworkObjectMode::Standalone);
     }
     {
         Node* node = Tests::SpawnOnServer<BehaviorNetworkObject>(serverScene, prefab, "Owned Node 1");
 
-        auto object = node->GetDerivedComponent<NetworkObject>();
+        auto object = node->FindComponent<NetworkObject>(ComponentSearchFlag::Self | ComponentSearchFlag::Derived);
```

Comment:
GetComponent and GetDerivedComponent are basically the same thing, I would expect API to be symmetrical for those. Having `GetComponent<T>()` and `FindComponent<NetworkObject>(ComponentSearchFlag::Self | ComponentSearchFlag::Derived)` is meh.